<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* Set background colors and properties */
QFrame#mainFrame{
background-color: #f3f4f7;
}

QFrame#contentFrame{
background-color: #fff;
}

QFrame#leftMenuFrame{
background-color: #36647b;
}

QWidget#page_License {
background-color: #f0f0f0;
}


QCheckBox{
font:  &quot;Open Sans&quot;;
font-weight: bold;
font-size: 8pt;
color: #D7DBEC;
}

QTextBrowser{
 font: 12pt &quot;Open Sans&quot;;
color:#171725;
background-color: #F7F8F9;
}
QPushButton{
background-color:#103782;
border-radius: 5px;
border:2px solid #103782;
height:40px;
width:150px;
font: 12pt &quot;Open Sans&quot;;
color:#FFFFFF;
}

QPushButton:hover{
background-color:#1F62E2;
border:2px solid#1F62E2;
}
QPushButton:checked{

background-color:#103782;
}
/* Styles for Tab Widgets */
QTabWidget::pane {
border:1px solid lightgray;
/*background: rgb(245, 245, 245);*/
} 

/* QLabel settings */
QLabel#SettingBg1{
border-radius: 5px;
border-style: solid;
border-width: 2px;
border-color: rgba(40, 40, 40, 0.2);
}

QLabel#SettingBg2{
border-radius: 5px;
border-style: solid;
border-width: 2px;
border-color: rgba(40, 40, 40, 0.2);
}

QLabel#SettingBg3{
border-radius: 5px;
border-style: solid;
border-width: 2px;
border-color: rgba(40, 40, 40, 0.2);
}

QLabel#SettingBg4{
font: 14pt &quot;Open Sans&quot;;
border-radius: 5px;
border-style: Semibold;
border-width: 2px;
border-color:  rgba(40, 40, 40, 0.2);
}

QLabel {
font-size: 12pt;
font:75  &quot;Open Sans&quot;;
font-weight: Semibold;
color:#171725;
background-color: #ECF1F7;
/*font-family: Noto Sans;*/
}

QTableView{
color:#171725;
background-color: #F7F8F9;

}

QHeaderView::section { 
font: 75 10pt &quot;Open Sans&quot;;
color: white;
background-color:#989898; 
}

QDateTimeEdit{
color:#171725;
background-color: #F7F8F9;
}

QTableView::item { 
font: 75 12pt &quot;Open Sans&quot;;
color:white; 
background:#2f373d; 
font-weight:900; 
}

QTableView::item:selected {
font: 75 12pt &quot;open Sans&quot;; 
color:black; 
background:#7cc8b9; 
font-weight:900; 
}

QTextEdit{
color:#171725;
background-color: #F7F8F9;
font: 10pt &quot;Open Sans&quot;;
}


QWidget{
background-color: #F7F8F9;
}
</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QTextEdit" name="connStatus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>801</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #F7F8F9;</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>801</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color: #ECF1F7;
}

</string>
     </property>
     <widget class="QPushButton" name="btnConnectDb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>154</width>
        <height>44</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:#103782;
border-radius: 5px;
border:2px solid #103782;
height:40px;
width:150px;
font: 12pt &quot;Open Sans&quot;;
color:#FFFFFF;
}

QPushButton:hover{
background-color:#1F62E2;
border:2px solid#1F62E2;
}
QPushButton:checked{
border:2px solid #103782;
background-color:#103782;
}</string>
      </property>
      <property name="text">
       <string>Connect DB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Welcome To PLCInsight Pro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="activateLicense">
      <property name="geometry">
       <rect>
        <x>614</x>
        <y>332</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:#103782;
border-radius: 5px;
border:2px solid #103782;
height:40px;
width:150px;
font: 12pt &quot;Open Sans&quot;;
color:#FFFFFF;
}

QPushButton:hover{
background-color:#1F62E2;
border:2px solid#1F62E2;
}
QPushButton:checked{
border:2px solid #103782;
background-color:#103782;
}</string>
      </property>
      <property name="text">
       <string>Activate</string>
      </property>
     </widget>
     <widget class="QLabel" name="licenseLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter License Key:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="licenseEnter">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>332</y>
        <width>409</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color :#ECF1F7;
</string>
      </property>
      <property name="text">
       <string>Click on connect to Logging</string>
      </property>
     </widget>
    </widget>
    <zorder>widget_2</zorder>
    <zorder>connStatus</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
